code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2472,6 +2472,56 @@
 		T.health -= 20
 		T.mutation_mod += 0.1
 
+/datum/reagent/toxin/insecticide
+	name = "Insecticide"
+	id = INSECTICIDE
+	description = "A broad pesticide. Do not ingest!"
+	reagent_state = REAGENT_STATE_LIQUID
+	color = "#49002E" //rgb: 73, 0, 46
+	density = 1.08
+	specheatcap = 4.18
+
+/datum/reagent/toxin/insecticide/reaction_mob(var/mob/living/M, var/method = TOUCH, var/volume)
+
+	if(..())
+		return 1
+	if(iscarbon(M))
+		var/mob/living/carbon/C = M
+		if(!C.wear_mask) //If not wearing a mask
+			C.adjustToxLoss(REM) //4 toxic damage per application, doubled for some reason
+		if(isinsectoid(C) || istype(C, /mob/living/carbon/monkey/roach)) //Insecticide being poisonous to bugmen, who'd've thunk
+			M.adjustToxLoss(10 * REM)
+					
+/datum/reagent/toxin/insecticide/reaction_animal(var/mob/living/simple_animal/M, var/method=TOUCH, var/volume)
+	if(..())
+		return 1
+		
+	if(istype(M,/mob/living/simple_animal/hostile/bigroach))
+		M.gib()
+		
+	if(istype(M,/mob/living/simple_animal/cockroach))
+		M.gib()
+	
+	if(istype(M,/mob/living/simple_animal/bee))
+		M.gib()
+
+	if(istype(M,/mob/living/simple_animal/hostile/giant_spider))
+		M.gib()
+	
+/datum/reagent/toxin/insecticide/reaction_obj(var/obj/O, var/volume)
+
+	if(..())
+		return 1
+	if(istype(O,/obj/machinery/portable_atmospherics/hydroponics))
```

Comment:
Why even have this if you have `on_plant_life()` below? Confusing why putting pest spray in and out repeatedly with a dropper instead of letting it absorb would allow you to quickly drop the pest level.

Reply:
Jury rigged. Should I delete it?

Reply:
Yes

Reply:
Deleted.